experimental/fRIP/fRIP_MPRNA_LibraryWorkupProtocol.md

```diff
@@ -0,0 +1,115 @@
+**Protocol typed by Evan Hass**  
+**Last modified on 2020-05-05**
+
+# DNase treatment of RNA  
+1. Aliquot 200 ng of RNA from each sample into PCR tubes, and bring to 16 uL with nuclease-free water. If a given sample is too dilute for 16 uL to contain 200 ng of RNA, just use 16 uL.  
+2. Make DNase master mix (1x recipe):  
+
+- 2 uL 10x Worthington DNase I Buffer  
```

Comment:
Is this the latest DNAse? Is the sigma roche version not the one to go with?

Reply:
The Worthington DNase I was called for in the MPRNA protocol we got from Harvard. I've not tried swapping that out for the sigma/roche combo.